subscription-engine/kafka-ws/bin/entryPoint.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+
+# -e Exit imediatly if a comand exit with non-zero status
+# -x Print commands and their arguments as they are executed
+
+# Debug mode
+if [ ! -z "${DEBUG+x}" ]; then
+    set -x
+fi
+
+export KAKKA_WS_CONNECTION_RETRY_COUNT=${KAFKA_WS_APP_CONNECTION_RETRY_COUNT:-"5"}
+export KAFKA_WS_CONNECTION_RETRY_TIMEOUT=${KAFKA_WS_CONNECTION_RETRY_TIMEOUT:-"3"}
+
+export KAFKA_WS_CONSUMER_METADATA_BROKER_LIST=${KAFKA_WS_CONSUMER_METADATA_BROKER_LIST:-"kafka:9092"}
+
+# Redis parameters
```

Comment:
I think it is better to put these settings close to the Redis' health check  code.